Neos.Neos/Resources/Private/Styles/_Fonts.scss

```diff
@@ -2,26 +2,26 @@
 	font-family: 'Noto Sans';
 	font-style: normal;
 	font-weight: 400;
-	src: local('Noto Sans'), local('NotoSans'), url('../Fonts/NotoSans/NotoSans-Regular.ttf') format('truetype');
+	src: local('Noto Sans'), local('NotoSans'), url('../Fonts/NotoSans/NotoSans-Regular.woff') format("woff");
```

Comment:
Did you try woff2 and see if they are even better?

Reply:
Did not try it but it should be bit smaller.

```
WOFF2 is the next generation of WOFF. The WOFF2 format offers a 30% average compression gain over the original WOFF. Because it still just a recommended upgrade, it does not have the wide support of WOFF.
```

But we would loose IE users https://caniuse.com/?search=woff2
So if we use it we need to include both. Like font-awesome does.